ui/pages/Search/components/VariantSearchFormContent.jsx

```diff
@@ -181,7 +206,22 @@ const PATHOGENICITY_PANEL = pathogenicityPanel(false)
 
 const ANNOTATION_PANEL = {
   name: 'annotations',
-  headerProps: { title: 'Annotations', inputProps: JsonSelectProps(ANNOTATION_FILTER_OPTIONS) },
+  headerProps: {
+    inputProps: {
+      component: Select,
+      options: ANNOTATION_FILTER_OPTIONS,
+      format: (val) => {
```

Comment:
Given that you have essentially-copy-pasted a lot of shared functionality for all the panels, you should instead update the `JsonSelectProps` helper to do this logic for you